presto-test-jdbc-compatibility/src/test/java/io/prestosql/TestJdbcCompatibility.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql;
+
+import io.airlift.log.Logging;
+import io.prestosql.jdbc.PrestoDriver;
+import io.prestosql.server.testing.TestingPrestoServer;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.Timestamp;
+
+import static com.google.common.base.Strings.repeat;
+import static java.lang.String.format;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestJdbcCompatibility
```

Comment:
Can you add a test that verifies that Presto Server version is matching `${project.version}` so we could be sure that we are not using Presto server that comes with `presto-jdbc`

Reply:
Do presto have something like presto_version() function or sth similar?

Reply:
`select node_version from system.runtime.nodes`?

Reply:
It's fixed to "testversion"

Reply:
I don't see how it could be done on CI